src/errors/test-run/templates.js

```diff
@@ -96,6 +115,18 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.selectActionElementNotFoundError]: err => markup(err, `
+        The specified ${err.argumentTitle} selector does not match any element in the DOM tree.
+
+        ${err.getCallsiteMarkup()}
+    `),
+
+    [TYPE.selectActionElementIsInvisibleError]: err => markup(err, `
+        The element that matches the specified ${err.argumentTitle} selector is not visible.
```

Comment:
Same for this two errors and `dragDestination` errors. Can be compound into `actionAdditionalElementIsInvisible`